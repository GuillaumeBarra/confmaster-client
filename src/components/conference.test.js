import React from 'react'
import {render} from '@testing-library/react'
import Conference from './conference'
 
 it('should take a snapshot', () => {
    const { asFragment } = render(<Conference />)
    
    expect(asFragment(<Conference />)).toMatchSnapshot()
   })