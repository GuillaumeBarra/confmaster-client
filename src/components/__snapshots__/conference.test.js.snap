// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    style="padding-right: 20px;"
  >
    <div
      class="MuiBox-root MuiBox-root-1 sc-bdvvtL iQhVPH"
    >
      <div
        class="MuiBox-root MuiBox-root-2"
      >
        <div
          style="color: white;"
        />
      </div>
      <div
        style="padding: 10px;"
      />
      <div
        style="padding: 10px;"
      />
      <div
        style="padding: 10px;"
      >
        My Role: 
      </div>
      <div
        style="padding: 10px;"
      />
    </div>
  </div>
</DocumentFragment>
`;
